[2025-07-09T22:20:12.800+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-09T22:20:12.819+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: list_s3_files.list_files_in_s3_bucket manual__2025-07-09T22:20:11.975127+00:00 [queued]>
[2025-07-09T22:20:12.831+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: list_s3_files.list_files_in_s3_bucket manual__2025-07-09T22:20:11.975127+00:00 [queued]>
[2025-07-09T22:20:12.831+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-09T22:20:12.845+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): list_files_in_s3_bucket> on 2025-07-09 22:20:11.975127+00:00
[2025-07-09T22:20:12.852+0000] {standard_task_runner.py:72} INFO - Started process 232041 to run task
[2025-07-09T22:20:12.856+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'list_s3_files', 'list_files_in_s3_bucket', 'manual__2025-07-09T22:20:11.975127+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/aws.py', '--cfg-path', '/tmp/tmpojz_96ri']
[2025-07-09T22:20:12.859+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask list_files_in_s3_bucket
[2025-07-09T22:20:12.909+0000] {task_command.py:467} INFO - Running <TaskInstance: list_s3_files.list_files_in_s3_bucket manual__2025-07-09T22:20:11.975127+00:00 [running]> on host dd8713d3b969
[2025-07-09T22:20:12.999+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='list_s3_files' AIRFLOW_CTX_TASK_ID='list_files_in_s3_bucket' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T22:20:11.975127+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T22:20:11.975127+00:00'
[2025-07-09T22:20:13.000+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-09T22:20:13.203+0000] {base.py:84} INFO - Retrieving connection 'aws_s3'
[2025-07-09T22:20:13.204+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T22:20:17.485+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-09T22:20:17.500+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-09T22:20:17.501+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=list_s3_files, task_id=list_files_in_s3_bucket, run_id=manual__2025-07-09T22:20:11.975127+00:00, execution_date=20250709T222011, start_date=20250709T222012, end_date=20250709T222017
[2025-07-09T22:20:17.568+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-09T22:20:17.611+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T22:20:17.612+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
